### ISSUE TITLE
documentation: (require 'ein-loaddefs) needed also

### ISSUE THREAD

**Issue by rebcabin** [millejoh/emacs-ipython-notebook#49]
When I was trying to bring this up, i followed the markdown documentation, which says (require 'ein).  But I found that wasn't enough, I also had to (require 'ein-loaddefs).  I found by reading the source that ein.el only does (provide 'ein); I didn't see it having any way to do anything else.  Maybe this is my misunderstanding, maybe the documentation needs a little more.

**Comment by millejoh** [comment]
You can go into ein.el and undocument the line that calls `(require 'ein-loaddefs)`. The MELPA maintainers advised me a few months ago that line wasn't necessary/redundant for people running ein from MELPA.

Am I right in assuming you have cloned ein from the git repository? Note that the easiest way to get ein working is to download from MELPA. This is how I load ein:

```elisp

(require 'ein-mumamo)
(require 'ein-dev)
(ein:dev-require-all)
```

You are right that, though, that the documentation needs some updating.

**Comment by rebcabin** [comment]
I first tried loading from melpa, and it didn't work at all.  I went to the #emacs freenode irc chat room, asked for advice, and they sent me straight to your fork.  In fact, if you go there and type ,ein the chatbot will send you to your site.  I noticed that you have been committing much more recently than the original site, so I decided to gamble that yours was the most up-to-date one.  I did the manual install as an additional "pack" in emacs-live (my current emacs setup).  It took a little while to get it all going but it seems to be working now and "
